app/views/admin/AdministerUserModal.coffee

```diff
@@ -207,3 +212,38 @@ module.exports = class AdministerUserModal extends ModalView
         @render()
       ), 2000)
     null
+
+  onClickEditPrepaidsInfoButton: (e) ->
+    prepaidId=@$(e.target).data('prepaid-id')
+    @prepaidTableState[prepaidId] = 'editMode'
+    @renderSelectors('#'+prepaidId)
+
+  onClickCancelPrepaidInfoEditButton: (e) ->
+    @prepaidTableState[@$(e.target).data('prepaid-id')] = 'viewMode'
+    @renderSelectors('#'+@$(e.target).data('prepaid-id'))
+
+  onClickSavePrepaidInfo: (e) ->
+    prepaidId= @$(e.target).data('prepaid-id')  
+    prepaidStartDate= @$el.find('#'+'startDate-'+prepaidId).val()
+    prepaidEndDate= @$el.find('#'+'endDate-'+prepaidId).val()
+    prePaidTotalLicenses=@$el.find('#'+'totalLicenses-'+prepaidId).val()
+    @prepaids.each (prepaid) =>
+      if (prepaid.get('_id') == prepaidId) 
+        #validations
+        unless prepaidStartDate and prepaidEndDate and prePaidTotalLicenses
+          return
+        if(prepaidStartDate >= prepaidEndDate)
+          alert('End date cannot be on or before start date')
```

Comment:
It would be better to show error using 'noty' like other errors on this page. For example: https://github.com/codecombat/codecombat/blob/dc27b3d365009480f450c35b6b5c6cffcec4b026/app/views/admin/AdministerUserModal.coffee#L175